.github/renovate.json

```diff
@@ -0,0 +1,29 @@
+{
+  "extends": ["config:base", ":disableDependencyDashboard"],
+  "prConcurrentLimit": 3,
+  "branchConcurrentLimit": 3,
+  "automergeType": "branch",
```

Comment:
I wonder if we should start with the default `pr` value for [`automergeType`](https://docs.renovatebot.com/configuration-options/#automergetype). The `branch` option won't create PRs in most cases which might be nice for noise reduction but I'm a little hesitant to do away with them entirely. It'd be nice to hear other opinions.